iceoryx_posh/test/moduletests/test_roudi_portmanager_fixture.hpp

```diff
@@ -102,6 +106,15 @@ class PortManager_test : public Test
         delete m_roudiMemoryManager;
         iox::rp::BaseRelativePointer::unregisterAll();
     }
+
+    void addAllInternalPublisherToVector()
```

Comment:
The name is misleading since this adds only the port introspection publisher

Reply:
Fixed